### PR TITLE
PLATUI-714: Add additional containers and entrypoints

### PR DESCRIPTION
**Pull request by matthewmascord**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -2,8 +2,17 @@
...
-GET        /tracking-transitional.js controllers.Assets.at(path="/public", file="tracking-transitional.js")
+GET        /tracking.js              controllers.Assets.at(path="/public", file="a.js")
+GET        /tracking/a.js            controllers.Assets.at(path="/public", file="a.js")
+GET        /tracking/b.js            controllers.Assets.at(path="/public", file="b.js")
+GET        /tracking/c.js            controllers.Assets.at(path="/public", file="c.js")
+GET        /tracking/d.js            controllers.Assets.at(path="/public", file="d.js")
+GET        /tracking/e.js            controllers.Assets.at(path="/public", file="e.js")
+GET        /tracking/f.js            controllers.Assets.at(path="/public", file="f.js")
+GET        /tracking/sdes.js         controllers.Assets.at(path="/public", file="sdes.js")
+GET        /tracking/transitional.js controllers.Assets.at(path="/public", file="transitional.js")
```
Comment (JoPintoPaul) [inline conf/app.routes:13]: I see we have two endpoints in the routes for transitional, is this deliberate to support people who have already integrated?

File: app/uk/gov/hmrc/trackingconsentfrontend/assets/src/constants/gtm.ts [hunk 2]
```diff
@@ -1,2 +1,8 @@
-export const DEFAULT_CONTAINER_ID = 'GTM-NDJKHWK'
 export const TRANSITIONAL_CONTAINER_ID = 'GTM-TSFTCWZ'
```
[the remark this answers is not in the document]
Comment (matthewmascord) [reply app/uk/gov/hmrc/trackingconsentfrontend/assets/src/constants/gtm.ts:1]: Yes.

File: app/uk/gov/hmrc/trackingconsentfrontend/assets/src/entrypoints/transitional.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import '../common/createNonce'
```
[the remark this answers is not in the document]
Comment (matthewmascord) [reply app/uk/gov/hmrc/trackingconsentfrontend/assets/src/entrypoints/transitional.ts:1]: 8 in total including the transitional container. They are all used in production.

File: conf/app.routes (hunk 1 above)
Comment (matthewmascord) [reply conf/app.routes:13]: Yes that is to keep supporting those until we can migrate them.

**Review by JoPintoPaul — APPROVED** [review]

**Pull request merged**